src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -230,17 +244,22 @@ def decrypt_secret_value(connection_name, encrypted_secret_value):
 
 
 def dump_yaml(*args, **kwargs):
```

Comment:
add some example in docstring on the possible calling pattern?

Reply:
Updated